Off-Top-App/off-top-java
# OT-97 -- Create a consumer in Java that will subscribe to the "outgoingResponse" topic from Python

# What was done
Created a consumer that listens to the topic `outgoingFocusScore`

# How to test
- Run spring-boot server
- Run zookeeper
Windows: `zookeeper-server-start.bat config\zookeeper.properties`
Mac: `zookeeper-server-start.sh config/zookeeper.properties`
 - Run Kafka
Windows: `kafka-server-start.bat config\server.properties`
Mac: `bin/kafka-server-start.sh config/server.properties`
- After Kafka is running properly create a producer in the command line
Windows: `windows\kafka-console-producer --broker-list localhost:9092 --topic outgoingFocusScore`
Mac: `bin/kafka-console-producer --broker-list localhost:9092 --topic outgoingFocusScore`

- Send a message like this in the producer ```{"user_id":"12", "analyized_at":"2020-04-12T10:15:30", "focus_score":"3"}```

- The response should look like this
![image](https://user-images.githubusercontent.com/28835909/80247165-72b9a300-8622-11ea-8bbd-15606331c079.png)
